### PR TITLE
Revolver sound changes.

### PR DESCRIPTION
**Pull request by pireamaineach**
Back with another tiny bike-shedding P.R.

The main aspect of this P.R. are the new revolver sounds, something I wanted to gauge people's opinions on. They're recordings of actual guns (and also quite loud) rather than the current cartoon-ish sound effects. While some aren't quite the way I'd like them yet, specifically the .38 fire sound, I think it actually sounds rather nice, though I can see this being a controversial opinion.
If people end up liking it I figure I'm going to slowly work on doing this with all other kinds of guns, most notably automatic pistols.

:cl:
tweak: Made the revolver spin and fire sounds louder.
tweak: Made the Russian revolver use the nagant sprite.
tweak: Made the golden revolver use the .357 fire sound instead of the resonator blast sound.
soundadd: Adds new revolver sounds, including spinning loading, firing, and ejecting sounds.
sounddel: Entirely removes all old revolver sounds, including the spinning, loading, firing, and ejecting sounds.
/:cl:

### CODE REVIEW

**Comment by pireamaineach** [comment]
Jesus fuck I made a mistake.
That's a lot a tags.

**Comment by Gousaid67** [comment]
thats alot of changes in a single PR, those alone would already be 3 PRs of their own. You should probably close this PR and open each major idea seperatly, one for the balance, one for the resprites and the last one for the sounds.

**Comment by pireamaineach** [comment]
> thats alot of changes in a single PR, those alone would already be 3 PRs of their own. You should probably close this PR and open each major idea seperatly, one for the balance, one for the resprites and the last one for the sounds.

I've already explained two of those and I refuse to make a whole fucking P.R. just to change the damage of .38 by 5 points.

**Comment by pireamaineach** [comment]
Shiz can you maybe tell me why you don't like it rather than just downvoting? Like the whole fucking purpose of this P.R. is?

**Comment by ShizCalev** [comment]
in real world applications, 10mm has more power then a standard .40 S&W (10mm = .40 cal when converting measurements, however the cartridges do differ in specific dimensions.) likewise, 10mm have less power then a .357 magnum (magnum being the descriptor for a .38 caliber with an elongated casing, which results in greater muzzle pressure due to the increase in grain count; and therefore, greater FPS/energy behind it), however the difference between the two are more or less negligible for AP applications.

38 special has a FPS of 980~, .40 ~ 1150 FPS, 10mm ~1200FPS, and 357 ~1440 FPS.

as such, the change to .38 specials to be equivalent to .40/10mm has no real logical basis aside from wanting the detective's shit to hit people harder.

edit: during typing this, you removed the buff, so i guess it doesn't matter now.

**Comment by ShizCalev** [comment]
just checking, were the sounds recorded by you, or are they from an archive/video?

**Comment by pireamaineach** [comment]
Everything but the fire sounds were recorded by me.
Also 10mm auto is more powerful than .357 magnum.

**Comment by ShizCalev** [comment]
@pireamaineach FBI ballistics testing disagrees with that.

**Comment by pireamaineach** [comment]
> @pireamaineach FBI ballistics testing disagrees with that.

What 10mm did they use and what .357 did they use?

**Comment by pireamaineach** [comment]
I think I see the problem. You're using ft/s and not ft x lbs.

**Comment by pireamaineach** [comment]
Velocity doesn't tell the whole story, 10mm is a much heavier bullet travelling just slower.

**Comment by pireamaineach** [comment]
"Excuse me would you please tell me what the problem is so I can seek to fix it amicably?"
"No, I dislike this P.R. It yucky. Very bad."

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	recent_spin = world.time + spin_delay
 
 	if(do_spin())
-		playsound(usr, "revolver_spin", 30, FALSE)
+		playsound(usr, "revolver_spin", 100, FALSE)
```
Comment (ShizCalev) [inline code/modules/projectiles/guns/ballistic/revolver.dm]: playsound(src

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — CHANGES_REQUESTED** [review]
downmix the new sounds to mono, please.

File: code/modules/projectiles/guns/ballistic/revolver.dm (hunk 1 above)
Comment (pireamaineach) [reply code/modules/projectiles/guns/ballistic/revolver.dm]: Er... do you want me to put playsound(src, "revolver_spin", 100) instead?
Not a coder, if you can't tell.

**Comment by pireamaineach** [comment]
> downmix the new sounds to mono, please.

Working on it, I'm going to go ahead and get a better .38 sound. As is the clicking of the cylinder rotating just takes you out of it, doesn't sound like it's coming from the gun.

**Comment by necromanceranne** [comment]
The revolver sprite is pretty old as I understand it. This one looks pretty cool, though I have a feeling people will confuse it for the .38 for a bit after this PR.

**Comment by actioninja** [comment]
give me the raw sound files where you got the sounds from and I'll noise cancel and normalize them.

**Comment by pireamaineach** [comment]
> give me the raw sound files where you got the sounds from and I'll noise cancel and normalize them.

Recordings, no unedited versions left.

**Comment by pireamaineach** [comment]
I do enjoy that I have 4 dislikes and nobody yet has said if it's the sprites or sound they don't like.

It's almost like I specifically stated that the sounds would be removed from the P.R. if you'd just comment your distaste instead of disliking it with no context.

**Comment by pireamaineach** [comment]
> downmix the new sounds to mono, please.

I downmixed the sounds.

**Comment by actioninja** [comment]
I'm not a big fan of the sounds, but I have a vested interesting being literally the person who made the old ones.
For the sprites, I like the basic one but the gold one is muddy and lacks definition.

**Comment by pireamaineach** [comment]
The gold one, as I've said, looks far better when not against a pure light or dark background.

**Comment by WJohn** [comment]
I much prefer our current revolver sprites.

And no, the gold one look terrible even against a grey background. They both need more contrast, but the gold one especially.

**Comment by pireamaineach** [comment]
> I much prefer our current revolver sprites.
> 
> And no, the gold one look terrible even against a grey background. They both need more contrast, but the gold one especially.

I'm going to have to disagree. I think the gold revolver looks proper in game. I think the problem here is just that gold itself looks like shit.

**Comment by pireamaineach** [comment]
Fuck it I'll see about the gold one.

**Comment by pireamaineach** [comment]
Revolver
Okay, there.

**Comment by pireamaineach** [comment]
Removed the sprite changes, someone's taking up the mantle and I figure people agree with them.

**Comment by pireamaineach** [comment]
Christ alive.

**Comment by Fikou** [comment]
the golden revolver is a reference to nuclear throne, should keep the orangey color at the bottom
image

**Pull request closed without merging**